docs/Incompatibilities.md

```diff
@@ -0,0 +1,129 @@
+Incompatibilities between SGX-LKL and x86-64 Linux
+==================================================
+
+SGX-LKL is a port of Linux to a paravirtualised SGX environment, using LKL as a porting layer.
+SGX-LKL runs x86-64 programs and these programs execute the same machine instructions as they would outside of an enclave.
+The kernel; however, runs in quite a different environment to ring 0 on an x86-64 CPU and this can lead to some important differences.
+
+Compatibility differences
+-------------------------
+
+Some features are not supported at all and so applications that depend on them will not work.
+
+### Single process
```

Comment:
The title seems weird given context. I am expecting a list of things to are not supported, and it says "single process" which is the limitation rather than a feature that isn't supported or  thing that won't work.